### PR TITLE
Update install instructions in README

### PR DESCRIPTION
**Pull request by brandonparsons**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ Questions? Ping me [@gavinjoyce](https://twitter.com/gavinjoyce)
 
 This is an Ember CLI addon, so all you need to do is
 
-`npm install ember-component-inbound-actions --save`
+`npm install --save-dev ember-component-inbound-actions`
```
Comment (GavinJoyce) [inline README.md]: can you update this to `npm install --save ember-component-inbound-actions`?

**Comment by GavinJoyce** [comment]
Thanks, I've just added one small suggestion

**Comment by brandonparsons** [comment]
@GavinJoyce  hmmm - I specifically put that as `--save-dev` as it is customary for Ember add ons to go into `devDependencies`. Your thoughts?

**Comment by GavinJoyce** [comment]
@brandonparsons I believe that we should use `--save` as this is a run time dependency for an app, not a build time dependency. https://ozthegreat.io/wordpress/stop-using-npm-wrong-save-dev

**Comment by brandonparsons** [comment]
@GavinJoyce The majority of ember addons use `devDependencies`, probably because ember cil is responsible for packaging the runtime files from the addons. I'm certainly not that knowledgable on this front however.

For example - note that the `ember install` command from ember cli uses `--save-dev`: https://github.com/ember-cli/ember-cli/blob/master/lib/commands/install-npm.js#L19

**Comment by GavinJoyce** [comment]
@brandonparsons TIL, thanks for the explanation. Perhaps we should use `ember install`?

**Comment by GavinJoyce** [comment]
oh, it's being removed too.

**Comment by brandonparsons** [comment]
np :)

**Pull request merged**